randovania/games/am2r/logic_database/Industrial Complex.txt

```diff
@@ -1108,7 +1108,19 @@ Extra - minimap_data: [{'x': 67, 'y': 29}, {'x': 67, 'y': 30}, {'x': 68, 'y': 28
   * Layers: default
   * 4-Tiles High Dock to Lower Factory Lava Lake Access/Horizontal Dock to Lower Factory Intersection
   > Horizontal Dock to Lower Factory Hub
-      Hijump With Power Grip Wall
+      Any of the following:
+          Space Jump or Can Use Spider Ball
```

Comment:
spider ball needs no room rando or mid air morph/bombs/spring or something like that.